### ISSUE TITLE
Support for rollup 3

### ISSUE THREAD

**Issue by dmail** [FredKSchott/rollup-plugin-polyfill-node#61]
This plugin is explicitely requesting rollup 2 resulting in an error when trying to use rollup 3 + this plugin

```console
npm ERR! Could not resolve dependency:
npm ERR! peer rollup@"^1.20.0 || ^2.0.0" from rollup-plugin-polyfill-node@0.10.2
```

Any plan to support rollup 3 soon?

**Comment by dmail** [comment]
"This repo is completely community-maintained. If you have an issue and know JavaScript, help out and submit a PR. This may be the only way to get your issue resolved!"

👌  https://github.com/FredKSchott/rollup-plugin-polyfill-node/pull/62

**Comment by StoneCypher** [comment]
![image](https://user-images.githubusercontent.com/77482/197016289-e2c97340-c186-4db1-9e8d-47ab6b6f85aa.png)

**Comment by danielsss** [comment]
+1

**Comment by MichaelDeBoey** [comment]
Closed by #62
